discovery/gossiper.go

```diff
@@ -2984,3 +2752,232 @@ func (d *AuthenticatedGossiper) handleNodeAnnouncement(nMsg *networkMsg,
 	// TODO(roasbeef): get rid of the above
 	return announcements, true
 }
+
+// handleChanAnnouncement processes a new channel announcement.
+func (d *AuthenticatedGossiper) handleChanAnnouncement(nMsg *networkMsg,
+	ann *lnwire.ChannelAnnouncement,
+	ops []batch.SchedulerOption) ([]networkMsg, bool) {
+
+	// We'll ignore any channel announcements that target any chain other
+	// than the set of chains we know of.
+	if !bytes.Equal(ann.ChainHash[:], d.cfg.ChainHash[:]) {
+		err := fmt.Errorf("ignoring ChannelAnnouncement from chain=%v"+
```

Comment:
nit: I think this fits in 80 characters

```
err := fmt.Errorf("ignoring ChannelAnnouncement from chain=%v, "+
    "gossiper on chain=%v", msg.ChainHash, d.cfg.ChainHash)
```

Reply:
on my IDE it is already 80 characters with tab size of 8, and this puts me at 82